src/tikz_export.jl

```diff
@@ -0,0 +1,153 @@
+export export_performance_profile_tikz
+
+"""
+    function export_performance_profile_tikz(T, filename; kwargs...)
+
+Export tikz figure of the performance profiles given by `T` in `filename`.
+
+## Arguments
+
+* `T :: Matrix{Float64}`: each column of `T` defines the performance data for a solver (smaller is better).
+  Failures on a given problem are represented by a negative value, an infinite value, or `NaN`.
+* `filename :: String` : path to the tikz exported file.
+
+## Keyword Arguments
+
+* `solvernames :: Vector{String} = []` : names of the solvers, should have as many elements as the number of columns of `T`. If empty, use the labels returned by `performance_profile_axis_labels`.
+* `xlim::AbstractFloat=10.` : size of the figure along the x axis. /!\\ the legend is added on the right hand side of the figure.
+* `ylim::AbstractFloat=10.` : size of the figure along the y axis.
+* `nxgrad::Int=5` : number of graduations on the x axis.
+* `nygrad::Int=5` : number of graduations on the y axis.
+* `grid::Bool=true` : display grid if true.
+* `colours::Vector{String} = []` : colours of the plots, should have as many elements as the number of columns of `T`.
+* `linestyles::Vector{String} = []` : line style (dashed, dotted, ...) of the plots, should have as many elements as the number of columns of `T`.
+* `linewidth::AbstractFloat = 1.0` : line with of the plots.
+* `xlabel::String = ""` : x axis label. If empty, uses the one returns by `performance_profile_axis_labels`.
+* `ylabel::String = ""` : x axis label. If empty, uses the one returns by `performance_profile_axis_labels`.
+
+Other keyword arguments are passed `performance_profile_data`.
+
+"""
+function export_performance_profile_tikz(
+  T::Matrix{Float64},
+  filename::String;
+  solvernames::Vector{String}=String[],
+  xlim::AbstractFloat=10.,
+  ylim::AbstractFloat=10.,
+  nxgrad::Int=5,
+  nygrad::Int=5,
+  grid::Bool=true,
+  # markers::Vector{S} = String[],
+  colours::Vector{String} = String[],
+  linestyles::Vector{String} = String[],
+  linewidth::AbstractFloat = 1.0,
+  xlabel::String = "",
+  ylabel::String = "",
+  kwargs...)
+
+
+
+  logscale = true
+  if haskey(kwargs,:logscale)
+    logscale = kwargs[:logscale]
+  end
```

Comment:
Why not have `logscale` as a keyword argument instead?

Reply:
@d-monnet you missed this probably

Reply:
`logscale` is already included in the kwargs that are passed to `performance_profile_data`.

Reply:
You can add in the keyword arguments of thé function and, then, when calling `performance_profile_data` you add `logscale = logscale`.
You construction seems unnecessary to me.

Reply:
You're wright, not sure what I was thinking about... This is fixed with the last commit !